_includes/head.html

```diff
@@ -1,11 +1,22 @@
 <head>
+
+    <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
+    <link rel="icon" type="image/png" href="./img/logo/logoOA_svg.png">
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width initial-scale=1" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-
-    <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
+    <meta name="keywords" content="foss, open source, organisations, 2018, technologists, astronomy,
+    OpenAstronomy, astrophysics, GSoC, sunpy, heliopy, astropy, projects, members, casacore,
+    JuliaAstro, yt-project, TimeLab, poliastra, PlasmaPy, Sherpa, IMS, chiantipy, Gnuastro" />
```

Comment:
The names of the orgs could be taken directly from the `members` variable. Otherwise we would have to update this once an again.
Something like:

```rb
{{ site.members | join: ', ' }}
```

Reply:
@dpshelio I took the names of the members from the members variable now.